src/components/UserDetailsTooltip/index.js

```diff
@@ -1,19 +1,35 @@
+import React from 'react';
 import PropTypes from 'prop-types';
+import {propTypes as userDetailsTooltipPropTypes, defaultProps as userDetailsTooltipDefaultProps} from './userDetailsTooltipPropTypes';
+import BaseUserDetailsTooltip from './BaseUserDetailsTooltip';
 
 const propTypes = {
-    /** Children to wrap with Tooltip. */
-    children: PropTypes.node.isRequired,
+    ...userDetailsTooltipPropTypes,
+    shouldRender: PropTypes.bool,
```

Comment:
Missing comment description for `shouldRender`

Reply:
Fixed!